models/reports/sql/generic/user-audit-report.sql

```diff
@@ -0,0 +1,28 @@
+select
+    user_name as "{{ translate_string('userName', 'User name') }}",
+    user_role as "{{ translate_string('userRole', 'User role') }}",
+    display_id as "{{ translate_string('patientDisplayId','Patient ID') }}",
+    patient_category as "{{ translate_string('patientCategory', 'Patient category') }}",
+    triage_category as "{{ translate_string('triageCategory', 'Triage category') }}",
+    facility as "{{ translate_string('facilityName', 'Facility') }}",
+    department as "{{ translate_string('departmentName', 'Department') }}",
+    location_group as "{{ translate_string('areaName', 'Area') }}",
```

Comment:
From the database perspective it is location group
```suggestion
    location_group as "{{ translate_string('locationGroupName', 'Area') }}",
```